tools/rvn/CommandLineApp.cs

```diff
@@ -342,7 +441,30 @@ private static int ExitWithError(string errMsg, CommandLineApplication cmd)
             cmd.ShowHelp();
             return 1;
         }
+        
+        private static CommandOption ConfigureModeOption(CommandLineApplication cmd)
+        {
+            return cmd.Option("-m|--mode", "Option to choose setup from either lets encrypt or importing certificate", CommandOptionType.SingleValue);
+        }
+        
+        private static CommandOption ConfigureSetupParameters(CommandLineApplication cmd)
+        {
+            return cmd.Option("-s|--setup-parameters", "call setup endpoints and obtain the setup package for setup up the cluster", CommandOptionType.SingleValue);
+        }
 
+        private static CommandOption ConfigurePackageOutputFile(CommandLineApplication cmd)
+        {
+            return cmd.Option("-o|--package-output-file", "default output file", CommandOptionType.SingleValue);
+        } 
+        private static CommandOption ConfigureCertPath(CommandLineApplication cmd)
+        {
+            return cmd.Option("--cert-path", "Certificate path", CommandOptionType.SingleValue);
+        }
+        
+        private static CommandOption ConfigureCertPassword(CommandLineApplication cmd)
+        {
+            return cmd.Option("-p|--password", "Certificate password", CommandOptionType.SingleValue);
```

Comment:
Passing passwords as CLI arguments is not considered safe usually - it gets into commands history, is visible on `tty`s etc. Would be nice to allow to provide an alternative way - add ability to pass it through an enviroment variable.

Reply:
tnx. I also removed the `-p`. It may confuse the user with `-p` as if it is the path to the `certificate`